src/template.html

```diff
@@ -6,8 +6,44 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <img id="webpackImg" alt="" />
-    <h1>Hello webpack!</h1>
-    <p>webpack works fine</p>
+    <main>
```

Comment:
- [ ] I suggest you change the name for `html` file from `template.html` to `index.html` - yes, it will not make an error in your app, but it is required 👍 I know, you can fix it easily 💯

Reply:
File changed from template.html to index.html